includes/connect/class-wc-stripe-connect.php

```diff
@@ -90,27 +90,7 @@ public function maybe_handle_redirect() {
 				wp_safe_redirect( esc_url_raw( remove_query_arg( [ 'wcs_stripe_state', 'wcs_stripe_code' ] ) ) );
 				exit;
 
```

Comment:
Nit: Could we remove this empty line?

Reply:
done [beefaa3](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2432/commits/beefaa31ebb5180127e882889800f65246bb4aae)